src/storage/storage.ts

```diff
@@ -699,6 +699,27 @@ export class Storage implements IStorage {
     }
   }
 
+  /**
+   * Get account path xprivkey if available.
+   *
+   * @param {string} pinCode
+   * @returns {Promise<string>}
+   */
+  async getAcctPathXPrivKey(pinCode: string): Promise<string> {
+    const accessData = await this._getValidAccessData();
+    if (accessData.acctPathKey === undefined) {
+      throw new Error('Private key is not present on this wallet.');
```

Comment:
The current implementation of the `getAcctPathXPrivKey` returns `null` when it's not able to get it from the `accessData`

I agree with refactoring it to throw an error instead, but we must remember to update the mobile-wallet accordingly [here](https://github.com/HathorNetwork/hathor-wallet-mobile/blob/25519b9d6e068270cb96f9480082e2f595dcd5ee/src/screens/PinScreen.js#L164)